charts/cbcontainers-agent/cbcontainers-agent-chart/values.yaml

```diff
@@ -0,0 +1,10 @@
+# accessToken is the API token used by the agent to communicate with the backend
+accessToken: ""
```

Comment:
Why do we need this if we have accessTokenSecretName?
I think it is less secure as well